library/src/methods/getDefaults/getDefaults.ts

```diff
@@ -26,32 +27,27 @@ export function getDefaults<
   TSchema extends SchemaWithMaybeDefault<
     BaseSchema | ObjectSchema<ObjectEntries, any> | TupleSchema<TupleItems, any>
   >,
->(schema: TSchema): DefaultValues<TSchema> {
-  // Create defaults variable
-  let defaults: any;
-
-  // If schema contains a default function, set its default value
+>(schema: TSchema): DefaultValues<TSchema> | undefined {
```

Comment:
Hello, as per the [documentation for `getDefaults`](https://valibot.dev/api/getDefaults/):

> The difference to [getDefault](https://valibot.dev/api/getDefault/) is that for objects and tuples without an explicit default value, this function recursively returns the default values of the subschemas instead of undefined.

Did that change?

Reply:
No, it should work just the same. All unit tests passed, so I do not expect any bugs, but maybe we are missing something. Please create an issue if you encounter any problems.

Reply:
Ok thanks. It just broke the type when I upgraded so I was wondering.

Reply:
Feel free to share your code if you thing something is wrong.